configs/nextflow.hopper.config

```diff
@@ -65,10 +65,10 @@ params {
   PHASTCONS         = "${refpath}/annotation_dbs/hg38.phastCons100way.bw"
   
   // DBVAR //
-  DBVAR_DEL         = "${refpath}/annotation_dbs/dbvar_deletions_pathogenic_nonredundant_hg38.bed"
-  DBVAR_DUP         = "${refpath}/annotation_dbs/dbvar_duplications_pathogenic_nonredundant_hg38.bed"
-  DBVAR_INS         = "${refpath}/annotation_dbs/dbvar_insertions_pathogenic_nonredundant_hg38.bed"
-  DBVAR_HEADERS     = "${refpath}/annotation_dbs/dbvar_headers.txt"
+  DBVAR_DEL         = "${refpath}/annotation_dbs/dbvar_deletions_pathogenic_nonredundant_hg38_acc.bed"
```

Comment:
I tried to cross-check the changes here. I see a new additional column. Is this the ClinVar accession ID? From where did you grab it? // Curious